jdk.gradle

```diff
@@ -178,14 +178,21 @@ def findJdkHome( File f )
 
 def getJdkDownloadUrl()
 {
-    def JdkInfoUrl = 'https://api.adoptopenjdk.net/v2/info/releases/openjdk11?'
+    def jdkInfoUrl = 'https://api.adoptopenjdk.net/v3/assets/feature_releases/8/ga?'
```

Comment:
We are using 11, not 8

Reply:
Damn, forgot to change this. Was testing it with version 8 mostly, since downloading 11 takes me ages for some reason.